app/javascript/scavenger_hunt/components/MicrosoftLoginButton/MicrosoftLoginButton.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import MicrosoftLogin from "react-microsoft-login";
+
+const SCAVENGER_HUNT_CLIENT_ID = '819d4ee7-63c4-4df1-aa7d-730b05e1fe7b';
```

Comment:
This is temporary and will be different. Since we are going to replace it later on, I guess it fine if it is public rn

Reply:
I am fine with it as long as we change the client ID before we ship it.